database/db-router.js

```diff
@@ -0,0 +1,82 @@
+// This router is designed to work with promises to seed mongo, postgres, or cassandra databases
+// It assumes you are using promises
+
+let MongoItem, PostgresItem, shutdownPostgres, CassandraItem;
+
+const defaultCB = () => {};
+
+const mongo = {
+  connection: MongoItem,
+  addOne: (record)=> {
+    return new Promise((res, rej) => res(MongoItem.create(record)));
+  },
+  addMany: (records)=> {
+    return new Promise((res, rej) => res(MongoItem.insertMany(records)));
+  },
+  deleteAll: ()=> {
+    return new Promise((res, rej) => res(MongoItem.deleteMany({})));
+  },
+  connect: ()=> {
+    return new Promise((res, rej) => res(MongoItem = require('./index.js')));
+  },
+  disconnect: () => {
+    return new Promise((res, rej) => res(MongoItem.db.close()));
+  },
+  getOne: (id) => {
+    return new Promise((res, rej) => res(MongoItem.find({ id: parseInt(id) })));
+  },
+  getAll: (id) => {
+    return new Promise((res, rej) => res(MongoItem.find({})));
+  },
+};
+
+const postgres = {
```

Comment:
This is really cool! Are you using this for just seeding? Looks like these would also be used for api calls.